jwst/scripts/asn_edit.py

```diff
@@ -27,26 +28,36 @@ def main():
 When removing files, the filename must be found in the association. If
 not, no change will be made to the file.
     """
-    parser = argparse.ArgumentParser(description=description_text,
-                                     formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser = argparse.ArgumentParser(
+        description=description_text,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
 
     arg_group = parser.add_mutually_exclusive_group()
-    arg_group.add_argument('-a', '--add', action='store_true',
-                           help= 'Add files to association')
-    arg_group.add_argument('-r', '--remove', action='store_true',
-                           help= 'Remove files to association')
+    arg_group.add_argument(
+        "-a", "--add", action="store_true", help="Add files to association"
```

Comment:
Very happy about reformatting, except the changing of quotes. Is there a reason?  Working with RSS, the extra shift key is a problem. Is this a `black` thing?

Reply:
yeah that was an artifact from `black`; I reverted the quote changes and rebased